### PR TITLE
Include paths in failed email

### LINKED ISSUE #180: Build failure email does not include part name.
When multiple parts fail there are specific ones which I know are important to address first but there's no way to know which link is which in the email.

screen shot 2015-08-24 at 5 49 47 pm

### PR DESCRIPTION
**Pull request by beeflamian**
screen shot 2016-03-30 at 14 38 04

This resolves #180

### CODE REVIEW

File: lib/git_blame.rb [hunk 1]
```diff
@@ -98,7 +98,7 @@ def lookup_git_names_and_emails(git_names_and_emails)
     def parse_git_changes(output)
       output.split("::!::").each_with_object([]) do |line, git_changes|
         commit_hash, author, commit_date, commit_message = line.chomp.split("|")
-        next if commit_hash.nil?
+        next if commit_hash.nil? || commit_message.nil?
```
Comment (beeflamian) [inline lib/git_blame.rb]: nil value of commit_message will fail next line and I saw exception in dev.

File: app/views/build_mailer/build_break_email.html.haml [hunk 2]
```diff
@@ -22,6 +22,7 @@
         %li
           = link_to("Part number #{failed_build_part.id}", repository_build_part_url(@build.repository, @build, failed_build_part))
           = failed_build_part_sentence(failed_build_part)
+        = "- #{failed_build_paths(failed_build_part)}"
```
Comment (robolson) [inline app/views/build_mailer/build_break_email.html.haml]: a bit of a nit but would you mind making this text smaller and maybe a shade of grey?

**Comment by beeflamian** [comment]
new version:
screen shot 2016-04-01 at 12 00 31

File: app/helpers/mail_helper.rb [hunk 3]
```diff
@@ -7,4 +7,16 @@ def failed_build_part_sentence(build_part)
     end
     str.html_safe
   end
+
+  def failed_build_paths(build_part)
+    paths = build_part.paths
+
+    str = if %w(spec).include? build_part.kind
```
Comment (robolson) [inline app/helpers/mail_helper.rb]: I think this might be backwards. Would make more sense as:

```
str = if build_part.kind.include?('spec')
```

**Pull request merged**